.codecov.yml

```diff
@@ -174,7 +174,7 @@ coverage:
         #
         # For python3, coverage should not be reduced compared to its base:
         #
-        target: auto
+        target: 60%
```

Comment:
I would suggest to reduce the scope of the python files which requires coverage(for example, if a python script is already fully covered by xenrt, we don't have to require coverage from UT again), while for those python scripts indeed require coverage from UT, is 60% enough?

Reply:
After a discussion with @gangj, we agreed on adding the unit test coverage to 80%. In the future, if there are cases where the scripts are not UTable or we can cover them by manual test or XenRT test, we can handle it by excluding them from the check.